docker-compose.yml

```
@@ -107,6 +107,180 @@ services:
     networks:
       - app
 
+  api:
+    build:
+      context: elixir
+      cache_from:
+        - type=registry,ref=us-east1-docker.pkg.dev/firezone-staging/cache/api:main
+      args:
+        APPLICATION_NAME: api
+    image: ${API_IMAGE:-us-east1-docker.pkg.dev/firezone-staging/cache/api}:${API_TAG:-main}
```

```suggestion
    image: ${API_IMAGE:-us-east1-docker.pkg.dev/firezone-staging/firezone/api}:${API_TAG:-main}
```

I think I typo'd these in a previous PR